common/types/testutil.go

```diff
@@ -66,7 +66,7 @@ func RandomNodeID() NodeID {
 
 // RandomBallotID generates a random BallotID for testing.
 func RandomBallotID() BallotID {
-	var b [hash20Length]byte // TODO(mafa): BallotIDSize is 32???
+	var b [Hash20Length]byte // TODO(mafa): BallotIDSize is 32???
```

Comment:
This is something that I don't understand. Afaik the BallotID is a Hash20 (actually a Hash32 where 12 bytes are cut off) but it is then encoded with 32 bytes - essentially causing every ballot to have 12 0-bytes in its ID?